app/templates/gentelella/admin/footer.html

```diff
@@ -68,12 +68,13 @@ <h4 class="footer-header">Use Open Event</h4>
                 <div class="col-md-3 col-sm-4 col-xs-8 dropup col-xs-offset-2 col-sm-offset-4 col-md-offset-0">
                     <div class="dropdown-toggle footer-dropdown" data-toggle="dropdown">
                         <i class="fa fa-globe"></i>
-                        <span class="dropdown-text" style="color: #D2D6DF;">Singapore</span>
+                        <span class="dropdown-text lang-header" style="color: #D2D6DF;">Singapore</span>
                         <span class="fa fa-chevron-down"></span>
                     </div>
                     <ul class="dropdown-menu">
-                        <li><a href="#" style="color:#969191;">Singapore</a></li>
-                        <li><a href="#" style="color:#969191;">Deutschland</a></li>
+                        <li><a href="#" style="color:#969191;" class="translate" id="en">Singapore</a></li>
+                        <li><a href="#" style="color:#969191;" class="translate" id="en">Deutschland</a></li>
+                        <li><a href="#" style="color:#969191;" class="translate" id="de">Germany</a></li>
```

Comment:
That is what I wanted to implement but instead of languages there were places already there in the dropdown, So I just added Germany as one more place. Will correct it now.